Orange/widgets/data/owpaintdata.py

```diff
@@ -762,6 +762,9 @@ class OWPaintData(OWWidget):
 
     graph_name = "plot"
 
+    class Error(OWWidget.Error):
+        sparse_not_supported = Msg("Sparse data is not supported.")
```

Comment:
Also, this can probably be just Warning (e.g. `Sparse input data is ignored. `) since the widget can function without it.